frontend/benefit/applicant/Dockerfile

```diff
@@ -0,0 +1,98 @@
+# =======================================
+FROM helsinkitest/node:14-slim as appbase
+# =======================================
+
+# Use non-root user
+USER appuser
+
+# Yarn
+ENV YARN_VERSION 1.22.10
+RUN yarn policies set-version $YARN_VERSION
+RUN yarn config set ignore-engines true
+
+WORKDIR /app
+
+# Install dependencies
+COPY --chown=appuser:appuser package.json yarn.lock lerna.json ./
+COPY --chown=appuser:appuser benefit/applicant/package.json benefit/applicant/package.json
+COPY --chown=appuser:appuser shared/package.json shared/package.json
+
+
+RUN yarn --frozen-lockfile --check-files && yarn cache clean --force
+
+# Copy all files
+COPY --chown=appuser:appuser . .
+
+# =============================
+FROM appbase as development
+# =============================
+
+# Use non-root user
+USER appuser
+WORKDIR /app
+
+# Copy all files
+COPY --chown=appuser:appuser . .
+# Bake package.json start command into the image
+CMD ["yarn", "dev"]
+
+# ===================================
+FROM appbase as staticbuilder
+# ===================================
+# Set environmental variables for frontend (used in CI)
+ARG NEXT_PUBLIC_BACKEND_URL
+
+# Use non-root user
+USER appuser
+WORKDIR /app
+
+# copy all files
+COPY --chown=appuser:appuser . .
+
+# Build application
+RUN yarn build
+
+# ==========================================
+FROM helsinkitest/node:12-slim AS production
+# ==========================================
+
+# Use non-root user
+USER appuser
```

Comment:
This was a bit altered in the meanwhile, try to apply same changes here: 
https://github.com/City-of-Helsinki/yjdh/pull/37/commits/d4f413a3334a4aee276fb860d86b785214dd7eff